src/main/java/Ryan.java

```diff
@@ -0,0 +1,308 @@
+import java.util.Scanner;
+
+public class Ryan {
+
+    private static final Scanner in = new Scanner(System.in);
```

Comment:
Since this is a constant variable, it should be named with the appropriate uppercase letters separated by underscores